### DISCUSSION TITLE
Password text field

### DISCUSSION THREAD

**Discussion by verma611** [TomSchimansky/CustomTkinter#728] [Announcements]
Can you make a password field using customtkiner, for example the instead of the password letters it displays bulletpoints to hide the password

**Comment by KeksTM** [comment]
Have you tried using Google?

For example: https://www.tutorialspoint.com/how-to-create-a-password-entry-field-using-tkinter

**Comment by verma611** [comment]
> Have you tried using Google?
> 
> For example: https://www.tutorialspoint.com/how-to-create-a-password-entry-field-using-tkinter

Yes... but there is no "show" argument for the text Feild. I have been using tkiner for a long time, but i recently heard about this

**Reply by Chiemerie1** [reply]
You can create a custom entry widget that will have display texts as "******" of any symbol of your choosing

**Comment by LHannah958** [comment]
there literally is a show argument. by text field I assume you mean Entry, in which case something like the following would work:
passwordbox=ctkinter.CTkEntry(placeholder_text="Password",show="*")

**Reply by renanale** [reply]
or you can also do after your first code: passwordbox.configure(show="*")
